pkg/controllers/scalablenodegroup/v1alpha1/suite_test.go

```diff
@@ -53,24 +54,76 @@ var _ = AfterSuite(func() {
 
 var _ = Describe("Examples", func() {
 	var ns *environment.Namespace
+	var defaultReplicaCount = ptr.Int32(3)
```

Comment:
This is never reassigned, so consider making this

defaultReplicas := 3
or 
defaultRepliacs := ptr.Int32(3)

I'd avoid relying on ptr if you can just use native go.

Reply:
What's the difference in these two declarations?
```
var defaultReplicas  = ptr.Int32(3)
defaultRepliacs := ptr.Int32(3)
```
IIUC, latter is just a short method of doing what we are doing with `var` statement, am I missing something?

Reply:
From the go docs `The := syntax is shorthand for declaring and initializing a variable, e.g. for var f string = "apple" in this case.`. Fine either way. I tend to use the shorthand.